middleware/jwt.go

```diff
@@ -225,6 +218,24 @@ func JWTWithConfig(config JWTConfig) echo.MiddlewareFunc {
 	}
 }
 
+// defaultKeyFetch returns a signing key of the given token.
+func (config *JWTConfig) defaultKeyFetch(t *jwt.Token) (interface{}, error) {
```

Comment:
Naming would be more consistent with `defaultKeyFunc`.

Reply:
will do that together with other comments. Waiting for your reply on the use case I was referring to.

Reply:
Sounds great. It's always easier to review with a real use case.
The approach is pretty flexible now, so seems fitting to me.